### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by fancyme**
I added an image with animation :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Slow Quit Apps
 
+![Example](https://im4.ezgif.com/tmp/ezgif-4-7099fc8ae7af.gif)
```
Comment (dteoh) [inline README.md]: I get a 404.

Better to add the file directly to the repo, and then change the link to:

`App screenshot`

Comment (fancyme) [reply README.md]: Example

https://s8.hostingkartinok.com/uploads/images/2019/08/961640d6bd8f90811933392fd2a24918.gif

Comment (fancyme) [reply README.md]: Sorry, I don't know how to do it right. )

Comment (dteoh) [reply README.md]: No worries, thanks for making a preview image! I have placed the image inside of the repo.

**Pull request merged**